### ISSUE TITLE
Support for multisite is broken

### ISSUE THREAD

**Issue by riasvdv** [ethercreative/simplemap#115]
### Description
The field doesn't support multisite.

### Steps to reproduce

1. Add an address in an entry in one site
2. Change it in the other site
3. It's also changed on the first site

**Comment by monachilada** [comment]
+1

**Comment by cole007** [comment]
Experienced similar problem - worked out a local fix but obviously needs rolling out to the codebase:

1. need to update `src/fields/MapField.php` to include the following in the `MapField` class:

```
    /**
    * @inheritdoc
    */
    public static function supportedTranslationMethods(): array
    {
        // Don't ever automatically propagate values to other sites.
	return [
	    self::TRANSLATION_METHOD_SITE,
        ];
    }
```

See, for example, https://github.com/craftcms/cms/blob/develop/src/fields/Matrix.php#L57

2. a translation setting is then required in the `src/templates/field-settings.twig` template. However, in the absence of this, you can directly update the settings for your field in the database by changing the `translationMethod` column to `site` for your field in the `fields/craft_fields` table.

Not sure if this is the best method but in the short-term has worked for me - will see if can sort out a pull request for this.

**Comment by chrisrowe** [comment]
+1

**Comment by Tam** [comment]
This should be fixed in the latest release ([3.4.0](https://github.com/ethercreative/simplemap/releases/tag/3.4.0)).
